js/debug/job_recommendation.js

```diff
@@ -0,0 +1,55 @@
+(function ($) {
+    var click = function(jobId, recommender){
+      if(!JOBCENTRE.capabilities.localStorage)
+          return;
+
+      var click = {'jobId': parseInt(jobId), 'recommender': recommender, 'timestamp': new Date().toISOString()};
+
+      var clickArray = JSON.parse(window.localStorage.getItem('recommendationInfo'));
+      if(clickArray == null)
+          clickArray = [];
```

Comment:
Instead of storing an array like this:

```
[{
    "jobId": 269891,
    "recommender": "MoreLikeThis",
    "timestamp": "2017-08-28T23:53:50.611Z"
}, {
    "jobId": 269716,
    "recommender": "MoreLikeThis",
    "timestamp": "2017-08-28T23:54:20.569Z"
}]
```

I suggest you use an object which acts as a hash table.  Finding and replacing would be much easier (and faster) this way:

```
{
    "269891": {
      "recommender": "MoreLikeThis",
      "timestamp": "2017-08-28T23:53:50.611Z"
  },
  "269716": {
      "recommender": "MoreLikeThis",
      "timestamp": "2017-08-28T23:54:20.569Z"
  }
}
```